### PR TITLE
Add PROJECT, PIPELINE, TRIGGER keywords

### PR DESCRIPTION
**Pull request by mikejholly**
Adds a few new keywords.

### CODE REVIEW

File: earthfile-mode.el [hunk 1]
```diff
@@ -47,7 +47,10 @@
 	  "LOCALLY"
 	  "COMMAND"
 	  "DO"
-	  "IMPORT")
+	  "IMPORT"
+      "PROJECT"
+      "PIPELINE"
+      "TRIGGER")
```
Comment (vladaionescu) [inline earthfile-mode.el:53]: Tabs vs spaces?

Comment (mikejholly) [reply earthfile-mode.el]: Fixed.

**Review by vladaionescu — APPROVED** [review]

**Comment by wingyplus** [comment]
@mikejholly do we any have reference for those syntaxes? Just curious what is it doing. :)

**Comment by mikejholly** [comment]
Nothing yet, aside from the parser/lexer. All 3 correspond to Cloud CI features. `PROJECT` specifies the organization and project one is operating on, and the `PIPELINE/TRIGGER` commands make a particular target a Cloud Pipeline (with a set of triggers). more documentation coming soon!

**Comment by wingyplus** [comment]
Thank you for your information. 🙇

**Pull request merged**